### PR TITLE
(138) Form submit buttons are always shown

### PR DESCRIPTION
**Pull request by dgmstuart**
We thought this was a GDS-ish pattern, but actually it's not, and leads
to complications e.g. when revisiting a form (an option is already
selected, but the button is not visible)

### CODE REVIEW

File: spec/features/resident_can_locate_problem_spec.rb [hunk 1]
```diff
@@ -100,7 +100,7 @@
     expect(page).to have_content('N1 6NU')
   end
 
-  scenario 'performing a search and not selecting an address, prevents form submission', js: true do
+  scenario 'performing a search and not selecting an address, prevents form submission' do
```
Comment (leeky) [inline spec/features/resident_can_locate_problem_spec.rb]: I think this test can be removed altogether - the non-JS path is tested by `performing a search and not selecting an address` below.

**Review by leeky — CHANGES_REQUESTED** [review]

File: spec/features/resident_can_locate_problem_spec.rb (hunk 1 above)
Comment (dgmstuart) [reply spec/features/resident_can_locate_problem_spec.rb]: Ah - sorry: I was too blasé and didn't actually read the test. Amended the original commit

**Comment by dgmstuart** [comment]
@leeky ready for another look

**Review by leeky — APPROVED** [review]

**Comment by leeky** [comment]
LGTM! :shipit:

**Pull request merged**